scripts/eval/rscc_grid_search_script.py

```
@@ -87,91 +88,110 @@ def main(args: argparse.Namespace):
-            # Validate extraction
-            if _extracted_computed is None or _extracted_computed.shape[0] == 0:
-                raise ValueError("Extracted computed map is empty")
+                # Calculate RSCC on extracted regions
+                # TODO: don't alter the input object _exp, just get a copy of it as a dict.
+                _exp.rscc = rscc(_extracted_base, _extracted_computed)
+                _exp.base_map_path = _base_map_path
 
-            # Calculate RSCC on extracted regions
-            _exp.rscc = rscc(_extracted_base, _extracted_computed)
-            _exp.base_map_path = _base_map_path
+            except Exception as _e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)